Zend/zend_strtod.c

```diff
@@ -3613,11 +3613,11 @@ rv_alloc(i) int i;
 rv_alloc(int i)
 #endif
 {
-	int j, k, *r;
+	int k, *r;
 
-	j = sizeof(ULong);
+	size_t j = sizeof(ULong);
 	for(k = 0;
-		sizeof(Bigint) - sizeof(ULong) - sizeof(int) + (size_t)j <= (size_t)i;
+		sizeof(Bigint) - sizeof(ULong) - sizeof(int) + j <= (size_t)i;
```

Comment:
Shouldn't `k` and `i` also be `size_t` ? Seems like this file could make use of a bit of cleanup to get rid of all the K&R definitions

Reply:
I thought of it for sure but :

- the issue is on `j` incrementation alone.
- it is a stable branch, I think the cleanup you mention is more appropriate for `master` branch (thinking if there is more implications from `int` => `size_t` changes, etc)

Reply:
Ah right this targets 8.2... yeah sure then do the minimal changes